styles.css

```diff
@@ -2,7 +2,7 @@
 
 body {
   font-family: 'Poppins', sans-serif;
-  background: rgb(255, 255, 255);
+  background: rgba(244, 238, 238, 0.829);
```

Comment:
- [ ] kindly use the same background color mentioned in Figma which is `#F7F7F9`